openerp/tools/config.py

```diff
@@ -443,6 +443,7 @@ def die(cond, msg):
             'test_file', 'test_enable', 'test_commit', 'test_report_directory',
             'osv_memory_count_limit', 'osv_memory_age_limit', 'max_cron_threads', 'unaccent',
             'data_dir',
+            'server_wide_modules',
```

Comment:
Cosmetically, you could join this entry with the entry of the line above. In OCB I'm a bit weary of conflicts between Odoo and OCB, and they are usually due to issues such as these ;-)